modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/socket/WebSocketManager.java

```diff
@@ -180,16 +195,62 @@ private void ping(WebSocketSession ws) {
     }
 
     /**
-     * @param evt Event to process.
      * @param ws Session.
+     * @param evt Event to process.
      * @throws IOException If failed to process.
      */
-    private void registerAgent(WebSocketEvent evt, WebSocketSession ws) throws IOException {
-        AgentInfo agentInfo = fromJson(evt.getPayload(), AgentInfo.class);
+    private void registerAgent(WebSocketSession ws, WebSocketEvent evt) throws IOException {
```

Comment:
Why this method receive `WebSocketSession` and `WebSocketEvent`? You can remove nested `if-else` if this method only receive `AgentHandshakeRequest` and return `AgentHandshakeResponse`